src/edit/LockHandle.js

```diff
@@ -12,6 +12,59 @@ L.LockHandle = L.EditHandle.extend({
 	_onHandleDrag: function() {
 	},
 
+  _bindListeners: function() {
+    this.on(
+      {
+        dragstart: this._onHandleDragStart,
```

Comment:
A snakecase `dragStart` would look great here

Reply:
It's actually a keyword (leaflet event) so its immutable.